R/driver-databricks.R

```diff
@@ -202,7 +215,19 @@ databricks_user_agent <- function() {
   user_agent
 }
 
-databricks_auth_args <- function(host, uid = NULL, pwd = NULL) {
+databricks_auth_args <- function(host, uid = NULL, pwd = NULL, session = NULL) {
+  # If a session is supplied, any viewer-based auth takes precedence.
+  if (!is.null(session)) {
+    access_token <- connect_viewer_token(session, paste0("https://", host))
```

Comment:
Should we emit something here? Seems like it would be useful to know when you've opted-in and it's not working. (And it wouldn't be too annoying to see that message when running locally.)

Reply:
Modified this so you see the following when running locally:
```
! Ignoring `sesssion` parameter.
ℹ Viewer-based credentials are only available when running on Connect.
```